tests/python/gaia-ui-tests/gaiatest/apps/base.py

```diff
@@ -149,6 +150,13 @@ def keyboard(self):
         from gaiatest.apps.keyboard.app import Keyboard
         return Keyboard(self.marionette)
 
+    @property
+    def manifest_url(self):
+        return '{}{}.{}/manifest.webapp'.format(self.DEFAULT_PROTOCOL, format(self.__class__).lower(), self.DEFAULT_APP_HOSTNAME)
```

Comment:
After that, I get errors related to the fact that entry_point doesn't exist. So I guess you also need to add that for every app. For apps that don't have it, it's an empty string, I guess.